modules/Package.swift

```diff
@@ -88,9 +89,11 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.5.4"),
         .package(url: "https://github.com/pointfreeco/swift-url-routing", from: "0.6.2"),
         .package(url: "https://github.com/zcash-hackworks/MnemonicSwift", from: "2.2.4"),
-        .package(url: "https://github.com/Electric-Coin-Company/zcash-swift-wallet-sdk", from: "2.2.3"),
+//        .package(url: "https://github.com/Electric-Coin-Company/zcash-swift-wallet-sdk", from: "2.2.3"),
```

Comment:
Should we rather remove this line?

Reply:
All commented code will be removed in a followup ticket (once other PRs are merged to the feature branch). In this specific case I need to share builds with Core team with the note commitment tree fix so it's going to be switched back to main SDK (2.2.4) with the fix.